verification/cbmc/stubs/aws_hash_table_no_slots_override.c

```diff
@@ -65,7 +65,11 @@ void make_hash_table_with_no_backing_store(struct aws_hash_table *map, size_t ma
      * CBMC will detect any attempt to use the slots.  This ensures that no code will ever accidentally use the values
      * in the slots, ensuring maximal nondeterminism.
      */
+    if (map == NULL) {
+        return;
+    }
     map->p_impl = bounded_malloc(sizeof(struct hash_table_state));
+    __CPROVER_assume(map->impl);
```

Comment:
Shouldn't this be `map->p_impl`

Reply:
```c
__CPROVER_assume(map->p_impl != NULL);
```

Reply:
Thanks for catching that! Was moving it over from my changes in the submodule. Updated!